### ISSUE TITLE
unable to install ibm_db on windows 11

### ISSUE THREAD

**Issue by MandiraRoyMukherjee** [ibmdb/node-ibm_db#949]
after npm install when i run npm run debug,this error coming.

window-11,64 bit
node js version-18.16.1, 64 bit

[server-debug] Error: \\?\C:\Users\004QG0744\CERT\node_modules\ibm_db\build\Release\odbc_bindings.node is not a valid Win32 application.
[server-debug] \\?\C:\Users\004QG0744\CERT\node_modules\ibm_db\build\Release\odbc_bindings.node
[server-debug]     at Module._extensions..node (node:internal/modules/cjs/loader:1340:18)
[server-debug]     at Module.load (node:internal/modules/cjs/loader:1119:32)
[server-debug]     at Module._load (node:internal/modules/cjs/loader:960:12)
[server-debug]     at Module.require (node:internal/modules/cjs/loader:1143:19)
[server-debug]     at require (node:internal/modules/cjs/helpers:110:18)
[server-debug]     at bindings (C:\Users\004QG0744\CERT\node_modules\bindings\bindings.js:112:48)
[server-debug]     at Object.<anonymous> (C:\Users\004QG0744\CERT\node_modules\ibm_db\lib\odbc.js:57:31)
[server-debug]     at Module._compile (node:internal/modules/cjs/loader:1256:14)
[server-debug]     at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
[server-debug]     at Module.load (node:internal/modules/cjs/loader:1119:32)
[server-debug]     at Module._load (node:internal/modules/cjs/loader:960:12)
[server-debug]     at Module.require (node:internal/modules/cjs/loader:1143:19)
[server-debug]     at require (node:internal/modules/cjs/helpers:110:18)
[server-debug]     at Object.<anonymous> (C:\Users\004QG0744\CERT\skills-shared\Services\DB2Service.js:2:11)
[server-debug]     at Module._compile (node:internal/modules/cjs/loader:1256:14)
[server-debug]     at Module._extensions..js (node:internal/modules/cjs/loader:1310:10) {
[server-debug]   code: 'ERR_DLOPEN_FAILED'
[server-debug] }
[server-debug]
[server-debug] Node.js v18.16.1

[server-debug] [nodemon] app crashed - waiting for file changes before starting...

**Comment by bimalkjha** [comment]
@MandiraRoyMukherjee `npm run debug` is used to start the app in debug mode. Are you able to start the app in non-debug mode? Please share complete output of  below commands from command prompt and not partial:
```
cd C:\Users\004QG0744\CERT\node_modules\ibm_db
npm install
cd test
open config.testConnectionStrings.json and update database connection info in this file and save it.
node test-basic-test.js
```
If `node test-basic-test.js` works fine, means there is no issue with ibm_db. Thanks.

**Comment by bimalkjha** [comment]
Duplicate of issue #947.
